src/product-card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.body__card {
+  width: 200px;
+}
+
+.card__image-wrapper {
+  padding: 32px 19px 0;
+  margin-bottom: 40px;
+  text-align: center;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-text);
+  margin: 0;
+  padding: 0 16px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-text-price);
+  padding: 0 16px;
+  margin: 0;
+  margin-bottom: 16px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 16px;
```

Comment:
add paddings to the card block, to avoid duplication

Reply:
done